plugin.rb

```diff
@@ -39,18 +39,22 @@ class Engine < ::Rails::Engine
 
   GlobalFilter::Engine.routes.draw do
     put '/filter_tags/:tag/assign' => 'filter_tags#assign'
+    get '/filter_tags' => 'filter_tags#index'
```

Comment:
I probably missed something, but I don't see where this route is used in the app.

Reply:
This is used in  `assets/javascripts/discourse/routes/admin-plugins-filter-tag.js`
```
return ajax("/global_filter/filter_tags.json").then((model) => {
```